distributed/dashboard/components/scheduler.py

```diff
@@ -1783,7 +1783,7 @@ def __init__(self, scheduler, **kwargs):
         self.last = 0
         self.source = ColumnDataSource(
             {
-                "time": [time() - 20, time()],
+                "time": [time() - 60, time()],
```

Comment:
this is obviously unrelated. this slows down the stealing dashboard a bit. 20s rolling window is very fast